webpack.common.js

```diff
@@ -0,0 +1,134 @@
+
+if (fileSystem.existsSync(secretsPath)) {
+  alias["secrets"] = secretsPath;
+}
+
+var options = {
+  entry: {
+    app: path.join(__dirname, "src", "js", "app.js"),
+    popup: path.join(__dirname, "src", "js", "popup.js"),
+    options: path.join(__dirname, "src", "js", "options.js"),
+    background: path.join(__dirname, "src", "js", "background.js"),
+    content: path.join(__dirname, "src", "js", "content.js")
+  },
+  // excludeEntriesToHotReload: ["content"],
+  output: {
+    path: path.join(__dirname, "build"),
+    filename: "[name].bundle.js"
+  },
+  node: {
+    fs: 'empty',
+    child_process: 'empty',
+    net: 'empty'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader'
+        }
+      },
+      {
+        test: /\.(css)$/,
+        include: [path.resolve(__dirname, 'node_modules/preact-material-components')],
+        use: [
+          { loader: 'style-loader' },
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 1,
+              localIdentName: '[local]',
+              modules: true,
+              sourceMap: true,
+            }
+          },
+          { loader: 'postcss-loader' }
+        ]
+      },
+      {
+        test: /\.css$/,
+        use: [
+          { loader: 'style-loader' },
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 1,
+              localIdentName: '[name]__[local]___[hash:base64:5]',
+              modules: true,
+              sourceMap: true,
+            }
+          },
+          { loader: 'postcss-loader' }
+        ],
+        include: [path.resolve(__dirname, 'src/app')]
+      },
+      {
+        test: new RegExp('\.(' + fileExtensions.join('|') + ')$'),
+        exclude: /node_modules/,
+        use: {
+          loader: "file-loader",
+          options: {
+            name: '[name].[ext]'
+          }
+        }
+
+      },
+      {
+        test: /\.html$/,
+        loader: "html-loader",
+        exclude: /node_modules/
+      }
+    ]
+  },
+  resolve: {
+    modules: [
+      path.resolve(__dirname, "src/lib"),
+      path.resolve(__dirname, "node_modules"),
+      'node_modules'
+    ],
+    alias: alias
+  },
+  plugins: [
+    // expose and write the allowed env vars on the compiled bundle
+    new webpack.DefinePlugin({
+      "process.env.NODE_ENV": JSON.stringify(env.NODE_ENV)
+    }),
+    new HtmlWebpackPlugin({
+      template: path.join(__dirname, "src", "popup.html"),
+      filename: "popup.html",
+      chunks: ["popup"]
+    }),
+    new HtmlWebpackPlugin({
+      template: path.join(__dirname, "src", "options.html"),
+      filename: "options.html",
+      chunks: ["options"]
+    }),
+    new HtmlWebpackPlugin({
+      template: path.join(__dirname, "src", "background.html"),
+      filename: "background.html",
+      chunks: ["background"]
+    }),
```

Comment:
I believe we need this added https://github.com/CityOfZion/NeoLink/commit/05daccd704a4290abfd530f5da29fa7b1e2642fc#diff-11e9f7f953edc64ba14b0cc350ae7b9d

Reply:
I actually already have that in my local branch. I'm not sure why it isn't showing up in the diff.